mpyl-test-runner.py

```diff
@@ -17,15 +19,20 @@ class StepParam:
     project: Project
 
 
-def execute_step(proj: Project, stage: Stage) -> MplOutput:
+def execute_step(proj: Project, stage: Stage) -> StepResult:
     config = parse_config("config.yml")
     properties = parse_config("run_properties.yml")
     run_properties = RunProperties.from_configuration(run_properties=properties, config=config)
-    executor = Steps(get_dagster_logger(), run_properties)
-    step_output = executor.execute(stage, proj).output
-    if not step_output.success:
-        raise Failure(description=step_output.message)
-    return step_output
+    dagster_logger = get_dagster_logger()
+    executor = Steps(dagster_logger, run_properties)
+    step_result = executor.execute(stage, proj)
+    # message = f'Executed {stage.name} for {proj.name}'
+    # dagster_logger.info(message)
+    # output = MplOutput(success=True, message=message)
+    # step_result = StepResult(stage=stage, project=proj, output=output)
```

Comment:
remove comment, or add flag to skip execution

Reply:
Solved in [cd77e12](https://github.com/Vandebron/mpyl/pull/28/commits/cd77e12967b68e8e2a1116cdc69e0594e141d6b3)